webui/react/src/components/FilterForm/components/FilterGroup.tsx

```diff
@@ -90,26 +90,29 @@ const FilterGroup = ({
     },
   });
 
-  const menuItems: DropDownProps['menu'] = useMemo(() => {
-    const onItemClick: MenuProps['onClick'] = (e) => {
-      if (e.key === FormKind.Field || e.key === FormKind.Group) {
-        formStore.addChild(group.id, e.key);
+  const onItemClick = useMemo(
+    () => (key: string) => {
```

Comment:
please import `useCallback` instead

Reply:
oops, yeah that would be the right way to do it.  I was splitting up a `useMemo` and got carried away